machineconfiguration/v1/types_machineconfignode.go

```diff
@@ -120,6 +129,12 @@ type MachineConfigNodeStatus struct {
 	// configVersion describes the current and desired machine config version for this node.
 	// +optional
 	ConfigVersion *MachineConfigNodeStatusMachineConfigVersion `json:"configVersion,omitempty"`
+	// configImage is an optional field that reflects the current state of the image used by the node.
+	// When omitted, this means that the Image Mode feature is not being used and the node will be up to date with the specific current rendered config version for the nodes MachinePool.
+	// When specified, the Image Mode feature is enabled and the contents of this field show the observed state of the node image.
```

Comment:
And what happens to the `configVersion` field?

Reply:
I think it would be good to have something in here that captures that the configuration image provides a base and the that config in configVersion extends it